rq/cli/cli.py

```diff
@@ -62,7 +62,10 @@
     click.option('--path', '-P',
                  default='.',
                  help='Specify the import path.',
-                 multiple=True)
+                 multiple=True),
+    click.option('--serializer', '-S',
+                 default=None,
+                 help='Serializer to use, JSON or Pickle')
```

Comment:
Can we clarify this a bit to `Path to serializer, defaults to rq.serializers.DefaultSerializer`?

Reply:
Mind addressing this comment?

Reply:
This help message should also be changed @f0cker . Apart from this, I think this PR is ready to merge :)

Reply:
Done. Let me know if you need anything else.